mindmeld/app_manager.py

```diff
@@ -285,7 +292,8 @@ def _pre_nlp(self, params, verbose=False):
     def _post_dm(self, request, dm_response):
         # Append this item to the history, but don't recursively store history
         prev_request = DialogueResponder.to_json(dm_response)
-        prev_request.pop("history")
+        if "history" in prev_request:
```

Comment:
We can remove this line even if it fails, since it will fail in master as well

Reply:
Removed.